assets/styles/app.scss

```diff
@@ -2,6 +2,7 @@
 
 body {
     margin:0;
-    background-color: $wild-color;
-    color: $light;
+    background-color: #000014;
+    height: 100%;
```

Comment:
fixed height will cause overflow if the content exceeds 100%, and percentage is not best unit here:

    min-height: 100vh;